racktables/find_unpingable_machines.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+python3 get_unused_machines.py | while read fqdn; do
+    ping -c1 -W1 "${fqdn}" > /dev/null 2>&1 && echo "${fqdn} up"
```

Comment:
Or maybe even
```suggestion
    ping -c1 -W1 "${fqdn}" &> /dev/null && echo "${fqdn} up"
```
as a more readable way to redirect both to `/dev/null`?

Reply:
I always forget this syntax and I hate it (because I regularly background my programs without wanting it…) but I added it :)